docs/absolute.md

```diff
@@ -333,8 +334,9 @@ missionLength.toLocaleString();
 epoch = new Temporal.Absolute(0n);
 billion = Temporal.Absolute.fromEpochSeconds(1e9);
 billion.difference(epoch);  // => PT1000000000S
-billion.difference(epoch, { largestUnit: 'hours' })  // => PT277777H46M40S
-billion.difference(epoch, { largestUnit: 'days' })  // => P11574DT1H46M40S
+billion.difference(epoch, { largestUnit: 'hours' });        // => PT277777H46M40S
+billion.difference(epoch, { largestUnit: 'days' });         // => P11574DT1H46M40S
+billion.difference(epoch, { largestUnit: 'nanoseconds' });  // => overflow; throws RangeError
```

Comment:
Ditto. (and this will actually serialize as `PT1000000000S` due to the ISO format not supporting separate subsecond units)

Reply:
Updated